src/en-GB/news/2021/v15-2-10-octopus-released/index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "v15.2.10 Octopus released"
+date: "2021-03-18"
+---
+
+This is the 10th backport release in the Octopus series. We recommend users to update to this release.
+
+## Notable Changes[¶](#notable-changes "Permalink to this headline")
```

Comment:
`[¶](#notable-changes "Permalink to this headline")` causes an error.

@Thingee Do you expect the headling-level anchor behaviour in the new site?

If so we can look at implementing a `markdown-it-anchor` solution.

Reply:
Done